RiotSharp.Test/appsettings.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "ApiKey": "RGAPI-c53bdafc-a9a8-4580-a30a-2c54e9403aba",
+  "ApiKey": "RGAPI-8e560850-6842-4158-b9bf-977f7e10d8a4",
```

Comment:
you're gonna have to regenerate that :+1:

Reply:
@BenFradet The api key was reverted back to the previous value.